src/main/java/newbank/Configuration.java

```diff
@@ -6,4 +6,10 @@
 
     public static final String DEFAULT_CURRENCY = "GBP";
     public static final int MAX_ACCOUNTS = 5;
+
+    public static double MAX_MICROLOAN = 1000;
+    public static double ACCRUAL_RATE = 0.01;
+    public static int MICROLOAN_OFFER_EXPIRY_DAYS = 7;
+    public static int MICROLOAN_REQUEST_EXPIRY_DAYS = 7;
+    public static int MAX_REPAYMENT_PERIOD_DAYS = 730;
```

Comment:
Hmm, this one sounds very magical? :D

Reply:
Haha! 2 years (ignoring leap years). Probably be nice _not_ to ignore them, but it seemed like an annoying amount of work for something tiny

Reply:
Sounds good! Perhaps could be commented.